commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp

```diff
@@ -276,7 +230,7 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 </aui:script>
 
 <aui:script use="liferay-dynamic-select">
-	new Liferay.DynamicSelect(
+	Liferay.component("<portlet:namespace />countrySelects", new Liferay.DynamicSelect(
```

Comment:
`"<portlet:namespace />countrySelects"` should be quoted with single quotes, and `"<portlet:namespace />countrySelects"` and `new Liferay.DynamicSelect(` should both be on new lines (like the `Liferay.provide` statements above).

Reply:
https://in.liferay.com/web/global.engineering/wiki/-/wiki/Core+Development+Main/Peer+Reviews+Checklist#section-Peer+Reviews+Checklist-New+Lines